samples/wifi/provisioning/README.rst

```diff
@@ -26,25 +26,29 @@ The sample requires a smartphone (configurator) with Nordic Semiconductor's nRF
 Overview
 ********
 
-With this sample, you can provision a Wi-Fi® device that lacks input or output capability using the :ref:`wifi_prov_readme` library.
+With this sample, you can provision a Wi-Fi device that lacks input or output capability using the :ref:`wifi_prov_readme` library.
 The sample is divided into three parts:
 
 * Task and event handling component: Handles provisioning-related tasks and events.
-* Transport layer: Based on Bluetooth® Low Energy and exchanges information between the device and the nRF Wi-Fi Provisioner app.
+* Transport layer: Based on Bluetooth Low Energy and exchanges information between the device and the nRF Wi-Fi Provisioner app.
 * Configuration management component: Manages the provisioning data (in RAM and flash) accessed by multiple threads.
 
 Configuration
 *************
 
+You can adjust the behavior of the sample by changing the configuration below.
+
 Advertisement data update
-*************************
+=========================
 
 You can enable the :kconfig:option:`WIFI_PROV_ADV_DATA_UPDATE` Kconfig option to get the live update of the Wi-Fi status without setting up a Bluetooth connection.
 When periodic update is enabled, the update interval is controlled by the :kconfig:option:`WIFI_PROV_ADV_DATA_UPDATE_INTERVAL` Kconfig option.
 
 Features
 ********
 
+The sample implements features below to enhance the user experience.
+
 Advertisement
 =============
```

Comment:
I would argue that this subheading is also not needed here and can be removed.

Reply:
Will remove this as well